pkg/kritis/review/validating_transport.go

```diff
@@ -69,16 +69,24 @@ func (avt *AttestorValidatingTransport) GetValidatedAttestations(image string) (
 		return nil, err
 	}
 	for _, a := range attestations {
-		decoded_sig, err := base64.StdEncoding.DecodeString(a.Signature)
-		if err != nil {
-			glog.Infof("Cannot base64 decode signature: %v", err)
-			continue
+		if a.SignatureType == metadata.GenericSignatureType || a.SignatureType == metadata.UnknownSignatureType {
```

Comment:
if a.SignatureType != metadata.PgpSignatureType ?

Reply:
This statement is to make sure that we do not try to verify generic signature types or unknown signature types. Eventually this will be modified to just `if a.SignatureType == metadata.UnknownSignatureType` once we support Generic Signature types.